src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/BundleSlingInitialContentJarEntryExtractor.java

```diff
@@ -109,12 +109,6 @@ void extractAndAddToAssembler(@NotNull BundleSlingInitialContentExtractContext c
                     throw new IOException("Can not convert " + file + " to enhanced DocView format", e);
                 }
 
-                // remap CND files to make sure they are picked up by the NodeTypesEntryHandler
-                if (context.getNamespaceRegistry().getRegisteredCndSystemIds().contains(file.getName())) {
-                    contentPackageEntryPath = "/META-INF/vault/" + Text.getName(file.getName()) + ".cnd";
```

Comment:
Why was the approach to convert to cnd files in a package discarded?

Reply:
This doesn't seem to have an effect afaik but I guess im wrong, il put it back in then

Reply:
Either put into repo init statements or into content packages, not both!